templates/inspec/tests/integration/build/gcp-mm.tf

```diff
@@ -836,3 +836,21 @@ resource "google_compute_node_group" "inspec-node-group" {
   size = var.node_group["size"]
   node_template = "${google_compute_node_template.inspec-template.self_link}"
 }
+
+resource "google_spanner_instance" "spanner_instance" {
+  project      = "${var.gcp_project_id}"
+  config       = "${var.spannerinstance["config"]}"
+  name         = "${var.spannerinstance["name"]}"
+  display_name = "${var.spannerinstance["display_name"]}"
+  node_count   = "${var.spannerinstance["node_count"]}"
+  labels = {
+    "${var.spannerinstance["label_key"]}" = "${var.spannerinstance["label_value"]}"
+  }
+}
+
+resource "google_spanner_instance" "database" {
+  project      = "${var.gcp_project_id}"
+  instance     = "${google_spanner_instance.spanner_instance.name}"
+  name         = "${var.spannerdatabase["name"]}"
+  ddl          = "${split("\n", file("ddl.sql"))}"
```

Comment:
This file is empty in this commit.

I would prefer to just put the string in the config file directly, and not depend on another file. Probably add it as a string in `mm-attributes.yml` and then have `ddl` be an array with that value

Reply:
Taken care